azext_iot/tests/iothub/devices/test_hub_export_import.py

```diff
@@ -0,0 +1,486 @@
+
+
+class TestHubExportImport(IoTLiveScenarioTest):
+    def __init__(self, test_case):
+        super(TestHubExportImport, self).__init__(test_case)
+        self.dest_hub = settings.env.azext_iot_desthub or "test-hub-" + generate_generic_id()
+        self.dest_hub_rg = settings.env.azext_iot_destrg or settings.env.azext_iot_testrg
+
+        # create destination hub
+
+        if not settings.env.azext_iot_desthub:
+            self.create_hub(self.dest_hub, self.entity_rg)
+
+        self.dest_hub_cstring = self.cmd(
+            f"iot hub connection-string show -n {self.dest_hub} -g {self.dest_hub_rg}"
+        ).get_output_in_json()["connectionString"]
+
+        self.filename = generate_generic_id() + ".json"
+
+        self.create_hub_state()
+
+    def create_hub_state(self):
+
+        # make a configuration for the hub (applies to 0 devices, this is just to test the configuration settings)
+
+        labels = {generate_generic_id() : generate_generic_id(), generate_generic_id() : generate_generic_id()}
+        labels = json.dumps(labels)
+
+        metrics_path = os.path.dirname(__file__) + "\\..\\configurations\\test_config_generic_metrics.json"
+        content_path = os.path.dirname(__file__) + "\\..\\configurations\\test_adm_device_content.json"
+
+        self.kwargs["config_content"] = read_file_content(content_path)
+        self.kwargs["labels"] = labels
+        self.kwargs["metrics"] = read_file_content(metrics_path)
+
+        self.cmd(
+            "iot hub configuration create --config-id hubConfig -l {} --content '{}' --labels '{}' --priority {} --metrics '{}' \
+            --target-condition {}".format(self.connection_string, "{config_content}", "{labels}", random.randint(1, 10),
+                                          "{metrics}", "tags.bar=12")
+        )
+
+        # make a regular edge deployment
+        deployment1_path = os.path.dirname(__file__) + "\\..\\configurations\\test_edge_deployment.json"
+        self.kwargs["edge_content1"] = read_file_content(deployment1_path)
+        self.cmd(
+            "iot edge deployment create -d deployment1 -l {} --content '{}' --labels '{}' --priority {} --metrics '{}' \
+            --target-condition {}".format(self.connection_string, "{edge_content1}", "{labels}", random.randint(1, 10),
+                                          "{metrics}", "tags.bar=12")
+        )
+
+        # make a layered edge deployment
+        deployment2_path = os.path.dirname(__file__) + "\\..\\configurations\\test_edge_deployment_layered.json"
+        self.kwargs["edge_content2"] = read_file_content(deployment2_path)
+        self.cmd(
+            "iot edge deployment create -d deployment2 -l {} --content '{}' --labels '{}' --priority {} --metrics '{}' \
+            --target-condition {} --layered".format(self.connection_string, "{edge_content1}", "{labels}", random.randint(1, 10),
```

Comment:
you can put another enter after the ( for more pythonic (or at least consistent) spacing